### DISCUSSION TITLE
What does "0" numbers if you "expand" a filename in the sidebar mean?

### DISCUSSION THREAD

**Discussion by johnmapeson** [losnoco/Cog#400] [Q&A]
This app is great, but there are two things I cannot figure out. The first question:

What does those `0` numbers mean? 

<img width="181" alt="image" src="https://github.com/user-attachments/assets/426345c4-a5d0-42ae-9424-672239ece72a">

**Comment by kode54** [comment]
Sorry for taking so long to respond to this. MP4 has chapter support, and single-chapter files are not collapsed by the sidebar listing. Gapless playback support for AAC MP4 files requires these chapters, as they define the valid range of the AAC stream. It indicates how much delay to chop off the start, and how much to ignore on the end of the file.

I can look into adding support to the sidebar tree so that it hides chapters in files with only a single chapter present. Not sure how hard that would be to implement, though.
